styles/style.css

```diff
@@ -1,192 +1,313 @@
 /* ========== Importing goole fonts ================== */
 @import url('https://fonts.googleapis.com/css2?family=Montserrat:ital@1&display=swap');
 
-
```

Comment:
There is a horizontal scroll which means some content is overflowing from the container. Please fix the overflow as it is not considered good for the design.
![overflow](https://user-images.githubusercontent.com/35504149/133644228-57632eb4-6483-4f3e-9714-566e3efc9611.png)